templates/zerver/help/include/subscriptions.md

```diff
@@ -1,2 +1,2 @@
-1. [Find the relevant stream](/help/browse-and-join-streams#browse-streams) on
+1. [Find the relevant stream](/help/browse-and-subscribe-to-streams#browse-streams) on
```

Comment:
Not sure why the change was needed here, I think `join` would be clearer for a new user.